Skyra.Core/Cache/Models/Prompts/CorePromptStateMessage.cs

```diff
@@ -19,26 +19,16 @@ public CorePromptStateMessage(ulong authorId, ulong channelId, object context)
 		[JsonProperty("cid")]
 		public ulong ChannelId { get; }
 
-		[JsonProperty("ctx")]
-		public object Context { get; private set; }
-
-		[NotNull]
-		public ICorePromptState Patch([NotNull] ICorePromptState value)
-		{
-			Context = value.Context;
-			return this;
-		}
-
 		[NotNull]
 		public string ToKey()
 		{
 			return ToKey(ChannelId, AuthorId);
 		}
 
-		public async Task RunAsync([NotNull] CoreMessage message, [NotNull] CorePromptStateMessage state)
+		public async Task<TimeSpan?> RunAsync([NotNull] CoreMessage message, [NotNull] CorePromptStateMessage state)
 		{
-			await message.SendAsync(
-				$"Oi there m8 you had a prompt set up, I replied to ya. By the way you once said `{state.Context}`");
+			await Task.CompletedTask;
```

Comment:
What is this? awaiting Task.CompletedTask is basically a no op. It looks like this method is actually useless - considering that you're also returning null.

Reply:
It's an overriding method, that class is like a "base" I'll use later and then delete (this one).